media-sound/naspro/naspro-0.2.91.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="Architecture for Sound PROcessing"
+DESCRIPTION="NASPRO core is the portable runtime library at the bottom of \
+the NASPRO Sound PROcessing Architecture"
+HOMEPAGE="http://naspro.sourceforge.net/"
+COPYRIGHT="2007-2011, NASPRO Development Team"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://sf.net/projects/naspro/files/naspro/$portVersion/naspro-core-$portVersion.tar.bz2"
+CHECKSUM_SHA256="00d61a9a18aacb8d0fc3e21ae56f005484111a52bdc47ce64677d568908bfe2f"
+SOURCE_DIR="naspro-core-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="3.1.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	naspro$secondaryArchSuffix = $portVersion
+	lib:libnacore$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	naspro${secondaryArchSuffix}_devel = $portVersion
+	devel:libnacore$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	naspro$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage naspro$secondaryArchSuffix \
+	$libDir/libnacore.so.$libVersion
```

Comment:
I've seen the quotes passing by since a few weeks (...), is there a real difference in using them or not?

Reply:
It does not make any difference right now because these paths do not contain any special character (like tabs or white spaces) since these paths are all in a chroot.
But:
* if haikuporter's `--nochroot` implementation gets a fix, and
* if a user has choosen to install the haikuports tree in a path containing one or more  spaces or tabs, and
* if that user tries to build a recipe (which does not have the quotes) using the `--nochroot` switch

then haikuporter would fail to create the debuginfo package.
Okay, I know that it is very unlikely that the conditions above would all be met, but my rationale is that adding the quotes does not cost much :)

Now, I don't care _that_ much about the quotes. I only add them when the change does not create a bigger diff (I mean, bigger wrt the number of lines in the git log), than if I did not add them.

Remark: I'm OK with not adding the quotes. That's why I tend to write "_Suggestion_" ;)